nbs/directives.ipynb

```diff
@@ -0,0 +1,828 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "datetime.datetime(2022, 8, 16, 21, 3, 42, 444782)"
+      ]
+     },
+     "execution_count": null,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "#|exec_doc\n",
+    "datetime.datetime.now()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## `#|notest`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Ignore this cell during testing."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## `#|hide`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When you use this directive, you will not see the cell input or output. Given the following input:\n",
+    "\n",
+    "```python\n",
+    "#|hide\n",
+    "print('you will not see this')\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "you will not see this\n"
+     ]
+    }
+   ],
+   "source": [
+    "#|hide\n",
+    "print('you will not see this')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You will not see any output above.\n",
+    "\n",
+    "As opposed to:\n",
+    "```python\n",
+    "print('you will see this')\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "you will see this\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('you will see this')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## `#|eval: <true|false>`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Example:\n",
+    "\n",
+    "```python\n",
+    "#|eval: false\n",
+    "raise Exception(\"I'm not raised because I'm not run\")\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#|eval: false\n",
+    "raise Exception(\"I'm not raised because I'm not run\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Compared to:\n",
+    "\n",
+    "```python\n",
+    "#|eval: true\n",
+    "assert 1 + 1 == 2\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#|eval: true\n",
+    "assert 1 + 1 == 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## `#|echo: <true|false>`\n",
+    "\n",
+    "This makes sure that only the output of a code cell is shown, not its input.\n",
```

Comment:
This is a quarto directive - see comments below on `#|output`

Reply:
See comment: https://github.com/fastai/nbdev/pull/868#discussion_r948232459